python/tk_multi_loader/model_publishhistory.py

```diff
@@ -82,6 +82,7 @@ def load_data(self, sg_data):
         filters = [ ["project", "is", sg_data["project"] ],
                     ["name", "is", sg_data["name"] ],
                     ["entity", "is", sg_data["entity"] ],
+                    ["sg_status_list", "not_in", ["omt", "ip"]],
```

Comment:
That doesn't seem to be the case, since without this line the omt and ip statuses show up in the history. I can continue investigating why they're not usuing the same filters though.